.github/workflows/tests-unit-js.yml

```diff
@@ -66,10 +65,18 @@ jobs:
         env:
           PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: true
 
+      - name: Download AMP validator JS
+        id: amp_validator
+        run: |
+          cd $RUNNER_TEMP && curl -O -f -s -S https://cdn.ampproject.org/v0/validator_wasm.js
+          AMP_VALIDATOR_FILE="${RUNNER_TEMP}/validator_wasm.js"
+          echo "::set-output name=validator_file::$AMP_VALIDATOR_FILE"
```

Comment:
This is a 7.2mb file. Anyway we can cache this?

Reply:
You mean between runs or ..?

I don't see an issue with downloading this anew for every CI run, ensures we're always using the latest version.

We could perhaps cache it with a very short expiration time or something, but that wouldn't really achieve anything.